iac-extensions/arm/src/test/java/org/sonar/iac/arm/checks/PublicNetworkAccessCheckTest.java

```diff
@@ -178,11 +180,56 @@ void shouldRaiseNoIssuesForUnknownType() {
     verifyNoIssue("PublicNetworkAccessCheckTest/publicNetworkAccess-Simplifed/unknown-type.json", CHECK);
   }
 
-  private static String readTemplateAndReplace(String type) {
-    Path filePath = Paths.get("src", "test", "resources", "checks", "PublicNetworkAccessCheckTest", "publicNetworkAccess-Simplifed").resolve("template.json");
+  static Stream<String> shouldCheckRangePublicIPAddress() {
+    return Stream.of(
+      "Microsoft.DBForMySql/flexibleServers/firewallRules",
+      "Microsoft.DBForPostgreSql/flexibleServers/firewallRules",
+      "Microsoft.DBforMariaDB/servers/firewallRules",
+      "Microsoft.DBforMySQL/flexibleServers/firewallRules",
```

Comment:
It is missing a test on a resource not present in the list with a public range IP that does not raise an issue.

Reply:
good catch, I had it in mind but forgot about